components/prize_pool/wikis/apexlegends/prize_pool_custom.lua

```diff
@@ -30,7 +30,7 @@ function CustomPrizePool.run(frame)
 	return prizePool:build()
 end
 
-function CustomLpdbInjector:adjust(lpdbData, placement, opponent)
+function CustomLpdbInjector:adjust(lpdbData, placement)
```

Comment:
ide will not like this (passing 3 params to a function that only accepts 2)

Reply:
Adjust interface uses varargs